cmr-exchange/metadata-proxy/project.clj

```diff
@@ -86,7 +86,7 @@
                                      :system :system
                                      :default (complement :system)}}
              :docs {:dependencies [[gov.nasa.earthdata/codox-theme "1.0.0-SNAPSHOT"]]
-                    :plugins [[lein-codox "0.10.6"]
+                    :plugins [[lein-codox "0.10.7"]
```

Comment:
I had to build this in order to get rid of the exception in the build in ous, even though I didn't change anything in metadata-proxy.